test/test_genbank_sbol3_direct.py

```diff
@@ -15,96 +15,133 @@ class TestGenBankSBOL3(unittest.TestCase):
     # Create converter instance
     converter = GenBank_SBOL3_Converter()
 
-    def test_gbtosbol3_1(self):
-        """Test conversion of a simple genbank file with a single sequence"""
-        SAMPLE_GENBANK_FILE_1 = Path(__file__).parent / "test_files" / "BBa_J23101.gb"
-        SAMPLE_SBOL3_FILE_1 = (
-            Path(__file__).parent
-            / "test_files"
-            / "BBa_J23101_from_genbank_to_sbol3_direct.nt"
-        )
-        sbol3.set_namespace(TEST_NAMESPACE)
-        TEST_OUTPUT_SBOL3 = str(SAMPLE_SBOL3_FILE_1) + ".test"
+    def _test_genbank_to_sbol3(self, SAMPLE_SBOL3_FILE: Path, SAMPLE_GENBANK_FILE: Path):
+        """Helper method to test conversion of a given GenBank file to SBOL3 using new converter.
+        :param SAMPLE_SBOL3_FILE: Path of expected SBOL3 converted file
+        :param SAMPLE_GENBANK_FILE: Path of given GenBank file to convert
+        """
+        TEST_OUTPUT_SBOL3 = str(SAMPLE_SBOL3_FILE) + ".test"
         # Don't write to file for testing, we directly compare sbol documents
         test_output_sbol3 = self.converter.convert_genbank_to_sbol3(
-            str(SAMPLE_GENBANK_FILE_1),
+            str(SAMPLE_GENBANK_FILE),
             TEST_OUTPUT_SBOL3,
             namespace=TEST_NAMESPACE,
             write=False,
         )
         sbol3_file_1 = sbol3.Document()
         sbol3_file_1.read(
-            location=str(SAMPLE_SBOL3_FILE_1), file_format=sbol3.SORTED_NTRIPLES
+            location=str(SAMPLE_SBOL3_FILE), file_format=sbol3.SORTED_NTRIPLES
         )
         assert not doc_diff(
             test_output_sbol3, sbol3_file_1
-        ), f"Converted SBOL3 file: {TEST_OUTPUT_SBOL3} not identical to expected file: {SAMPLE_SBOL3_FILE_1}"
+        ), f"Converted SBOL3 file: {TEST_OUTPUT_SBOL3} not identical to expected file: {SAMPLE_SBOL3_FILE}"
+    
+    def _test_sbol3_to_genbank(self, SAMPLE_SBOL3_FILE: Path, SAMPLE_GENBANK_FILE: Path):
```

Comment:
Same as above regarding upper case or lower case. If the arguments are intentionally upper case a comment should explain why. Otherwise these should be lower case.

Reply:
Right, fixed this.